src/Compiler/FSharp.Compiler.Service.fsproj

```diff
@@ -244,6 +247,9 @@
     <Compile Include="$(FsYaccOutputFolder)pppars.fs">
       <Link>SyntaxTree\FsYaccOutput\pppars.fs</Link>
     </Compile>
+<!--    <Compile Include="$(FsYaccOutputFolder)pars.fsi">-->
```

Comment:
Presumably this shouldn't be here (or should be uncommented?)

Reply:
Ah, no that is from the other PR. Should be removed here.